src/scss/base/_typography.scss

```diff
@@ -28,6 +28,7 @@
         font-family: Aspira;
         font-style: normal;
         font-weight: 800;
+        font-display: swap;
         src: url('#{$font-path}/menulog/aspira-heavy.woff2') format('woff2'),
              url('#{$font-path}/menulog/aspira-heavy.woff') format('woff'),
              url('#{$font-path}/menulog/aspira-heavy.ttf') format('truetype');
```

Comment:
I don't think we need ttf support now.

Reply:
@kevinrodrigues I wondered about that too. It's not in scope for this ticket, but can remove the path reference to it if you wish?

Reply:
Not a biggy here tbh but it would be good to remove them in another ticket if they're not required anymore.